.ci/scripts/distribution/qa-testbench.sh

```diff
@@ -1,7 +1,50 @@
-#!/bin/sh -eux
+#!/bin/bash -eux
 
 chmod +x clients/go/cmd/zbctl/dist/zbctl
 
-alias zbctl="clients/go/cmd/zbctl/dist/zbctl"
+zbctl="clients/go/cmd/zbctl/dist/zbctl"
 
-zbctl create instance qa-protocol --variables "${QA_RUN_VARIABLES}"
+"${zbctl}" create instance external-tool-integration --variables "${QA_RUN_VARIABLES}"
+
+businessKey="${BUSINESS_KEY}"
+
+waiting=1
+
+echo "Waiting for result of $businessKey"
+
+while [ $waiting -eq 1 ]; do
+
+    "${zbctl}"  activate jobs "$businessKey" > activationresponse.txt 2>error.txt
+
+    key=$(jq -r '.key' < activationresponse.txt)
+
+    if [ -z "$key" ]; then
+        echo "Still waiting"
+        sleep 5m
+    else
+        echo "QA run completed"
+        waiting=0
+    fi
+done
+
+key=$(jq -r '.key' < activationresponse.txt)
+
+echo "Job key is: $key"
+
+variables=$(jq -r '.variables' < activationresponse.txt)
+
+echo "Job variables are: $variables"
+
+testResult=$(echo "$variables" | jq -r '.aggregatedTestResult')
+
+echo "Test result is: $testResult"
+
+"${zbctl}"  complete job "$key"
+
+if [ "$testResult" == "FAILED" ]; then
+  echo "Test failed"
```

Comment:
With the information we have here, can we link to the result page already? Could save you time if you can just immediately click to go and see your results. Here or in the build results, whichever.

Reply:
Which result page? Not sure what you mean?

Reply:
The `Test Reports (Zeebe Cluster Testbench)` Google doc - but I already noticed you mentioned we'll be pulling in the results in a separate PR, so that's moot :+1:

Reply:
@npepinpe Still no idea what you mean. And I have no plans to do much more than what is already there. Let's discuss that tomorrow. Maybe there was a misunderstanding or I missed somethin.

Reply:
Yeah, I was probably not explaining myself correctly.

I was thinking, what would I do when I see that the QA stage fails - where should I look, how can I know what went wrong? As far as I know, we have a shared spreadsheet in Google Docs which collects the test results. To correlate the results, I need to know the generation so I can look up the relevant tests in there. Once I know what went wrong, what else can I do? I can find the cluster ID and look up the logs in Google Cloud, or check Operate in Testbench.

I just thought, that's a lot of places and I need to know all of these - what if we added the links directly in the test results/build result? That way I can just click on it and be taken directly from Jenkins to the relevant places to figure out what went wrong.

But like I said, if you plan on already importing the results into Jenkins, then there's nothing to be done :+1: 

Hope that helps, otherwise let's quickly talk about it in our sync so I can improve how I communicate :slightly_smiling_face:

Reply:
Yeah, let's talk about it tomorrow. I don't plan to export test results into Jenkins - not in the way I think you imagine it. Which I realize now sounds confusing. 

So the only test result that Jenkins will know about is whether all tests succeeded or any test failed. This is done with this PR and I wanted to keep it at that,

But I understand your desire to make correlation easier. 

There is: https://github.com/zeebe-io/zeebe-cluster-testbench/pull/188
This will make it easier. But maybe we need a little more.